core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -763,6 +766,25 @@ bool core_mmu_find_table(struct mmu_partition *prtn, vaddr_t va,
 	return ret;
 }
 
+static uint64_t *core_mmu_xlat_table_alloc(struct mmu_partition *prtn)
+{
+	uint64_t *new_table;
+
+	if (prtn->xlat_tables_used >= MAX_XLAT_TABLES) {
+		DMSG("out of xlat tables to allocate! total is %u",
```

Comment:
No need for bang `!` character.
Suggestion: use `EMSG()` and with a short message like `EMSG("%u xlat tables exhausted", MAX_XLAT_TABLES);`